sunpy/timeseries/sources/__init__.py

```diff
@@ -1,9 +1,19 @@
-"""Datasource-specific classes
+"""
+This module provies a collection of datasource-specific
+`~sunpy.timeseries.TimeSeries` classes.
 
-This is where datasource specific logic is implemented. Each mission should
-have its own file with one or more classes defined. Typically, these classes
-will be subclasses of the :mod`sunpy.TimeSeries` class.
+Each mission should have its own file with one or more classes defined.
+Typically, these classes will be subclasses of the
+`sunpy.timeseries.TimeSeries`.
 """
-__all__ = ['rhessi', 'eve', 'goes', 'lyra', 'noaa', 'norh', 'fermi_gbm']
+from sunpy.timeseries.sources.eve import EVESpWxTimeSeries
+from sunpy.timeseries.sources.fermi_gbm import GBMSummaryTimeSeries
+from sunpy.timeseries.sources.goes import XRSTimeSeries
+from sunpy.timeseries.sources.lyra import LYRATimeSeries
+from sunpy.timeseries.sources.noaa import NOAAIndicesTimeSeries, NOAAPredictIndicesTimeSeries
+from sunpy.timeseries.sources.norh import NoRHTimeSeries
+from sunpy.timeseries.sources.rhessi import RHESSISummaryTimeSeries
 
-from . import eve, rhessi, goes, lyra, noaa, norh, fermi_gbm
+__all__ = ['EVESpWxTimeSeries', 'XRSTimeSeries', 'NOAAIndicesTimeSeries',
```

Comment:
Do we need this? We are not excluding anything from the namespace, so we probably don't really need it.